server/item/tool.go

```diff
@@ -84,3 +86,32 @@ func (n ToolNone) HarvestLevel() int { return 0 }
 
 // BaseMiningEfficiency ...
 func (n ToolNone) BaseMiningEfficiency(world.Block) float64 { return 1 }
+
+// toolTierRepairable returns true if the ToolTier passed is repairable.
+func toolTierRepairable(tier ToolTier) func(Stack) bool {
+	return func(stack Stack) bool {
+		switch tier {
+		case ToolTierWood:
+			if planks, ok := stack.Item().(interface{ Planks() bool }); ok {
+				return planks.Planks()
+			}
+		case ToolTierStone:
+			if cobblestone, ok := stack.Item().(interface{ Cobblestone() bool }); ok {
```

Comment:
I'd consider naming these methods differently, something to imply that they can repair wood tools. Maybe something like `RepairsWoodTools()` or something like that.